backend/curriculum_tracking/tests/test_activity_log_entry_creators.py

```diff
@@ -40,3 +84,102 @@ def test_that_only_one_entry_created_for_one_review(self):
+        self.assertTrue(exception, AttributeError)
+        self.assertEqual(LogEntry.objects.count(), 0)
+
+
+class log_project_and_topic_competence_reviews_done_TESTS(APITestCase, APITestCaseMixin):
+    LIST_URL_NAME = "agilecard-list"
+    SUPPRESS_TEST_POST_TO_CREATE = True
+    FIELDS_THAT_CAN_BE_FALSEY = [
+        "code_review_competent_since_last_review_request",
+        "code_review_excellent_since_last_review_request",
+        "code_review_red_flag_since_last_review_request",
+        "code_review_ny_competent_since_last_review_request",
+        "requires_cards",
+        "required_by_cards",
+        "project_submission_type_nice",
+        "topic_needs_review",
+        "topic_progress",
+        "due_time",
+        "complete_time",
+        "review_request_time",
+        "start_time",
+        "tag_names",
+        "can_start",
+        "can_force_start",
+        "flavour_names",
+        "open_pr_count",
+        "oldest_open_pr_updated_time",
+        "users_that_reviewed_since_last_review_request"
+    ]
+
+    def verbose_instance_factory(self):
+        project = RecruitProjectFactory()
+        card = AgileCardFactory(recruit_project=project)
+        return card
+
+    def setUp(self):
+        self.api_url = self.get_list_url()
+
+    @mock.patch.object(curriculum_tracking.activity_log_entry_creators, 'log_topic_competence_review_done')
+    @mock.patch.object(curriculum_tracking.activity_log_entry_creators, 'log_project_competence_review_done')
+    def test_log_project_competence_review_invoked_from_api_endpoint_for_project_review(
+            self, proj_comp_review, topic_comp_review
+    ):
+
+        super_user = UserFactory(is_superuser=True)
+        card = AgileCardFactory(content_item=ProjectContentItemFactory())
+        self.login(super_user)
+        response = self.client.post(
+                path=f'{self.api_url}{card.id}/add_review/',
+                data={"status": "NYC", "comments": "dammit"}
+            )
+
+        self.assertTrue(response.status_code, 200)
+        proj_comp_review.assert_called()
+        topic_comp_review.assert_not_called()
+
+    @mock.patch.object(curriculum_tracking.activity_log_entry_creators, 'log_topic_competence_review_done')
+    @mock.patch.object(curriculum_tracking.activity_log_entry_creators, 'log_project_competence_review_done')
+    def test_log_topic_competence_review_invoked_from_api_endpoint_for_topic_review(
+            self, proj_comp_review, topic_comp_review
+    ):
+        super_user = UserFactory(is_superuser=True)
+        card = AgileCardFactory(content_item=ContentItemFactory())
+        card.topic_progress = TopicProgressFactory()
+        card.save()
+        self.login(super_user)
+        response = self.client.post(
+            path=f'{self.api_url}{card.id}/add_review/',
+            data={"status": "NYC", "comments": "dammit"}
+        )
+
+        self.assertTrue(response.status_code, 200)
+        topic_comp_review.assert_called()
```

Comment:
What was it called with?

Also, What should the logentry look like? There is no test here that checks that the attributes are corect. Eg, what should object_1 be equal to? Who is the effected user? Etc? Very important that we make sure the entry is properly created.